tests/core/test_cost_calculation.py

```diff
@@ -131,7 +132,7 @@ async def test_mempool_mode(self, softfork_height, bt):
         pk = bytes.fromhex(
             "88bc9360319e7c54ab42e19e974288a2d7a817976f7633f4b43f36ce72074e59c4ab8ddac362202f3e366f0aebbb6280"
         )
-        puzzle = p2_delegated_puzzle_or_hidden_puzzle.puzzle_for_pk(pk)
+        puzzle = p2_delegated_puzzle_or_hidden_puzzle.puzzle_for_pk(G1Element.from_bytes(pk))
```

Comment:
We could hint to accept both.  We just `bytes()` down at the bottom which would end up doing nothing.